low-level/files/opt/seagate/sspl/conf/sspl.conf.LR2.yaml

```diff
@@ -180,6 +180,8 @@ SYSTEMDWATCHDOG:
    monitor: true
    threaded: true
    monitored_services:
+       - consul.service
```

Comment:
Any reason on why consul is added under monitored service in LR2?

Reply:
I have added this just for testing, This is not final list,  we will get final list from HA team.

Reply:
added services list.